index.html

```diff
@@ -10,19 +10,25 @@
     <header class="header">
         <div class="headerelement headerelement1">Welcome</div>
         <nav class="headerelement headerelement2"><img src="./Images/nav-menu1.png" height="25" width="25" alt="navimg"/></nav>
+        <nav class="header-desktop">
+            <div class="header-desktop1">Portfolio</div>
+            <div class="header-desktop2">About</div>
+            <div class="header-desktop3">Contact</div>
+        </nav>
     </header>
 
```

Comment:
-After ```<header>```, we can use ```<main>``` tags, that way, we have a clear division in our page, header, main, and footer

Reply:
Perfect, now I will better my Seo and semantic. it's done.